lib/UserAgentManager.php

```diff
@@ -40,7 +40,7 @@ public function __construct() {
 		$this->supportedUserAgents = [
 			'/^Mozilla\/5\.0 \(Android\) Nextcloud\-android.*$/' => '',
 			Request::USER_AGENT_CLIENT_DESKTOP => '',
```

Comment:
`Request::USER_AGENT_CLIENT_DESKTOP` resolves to `/^Mozilla\/5\.0 \([A-Za-z ]+\) (mirall|csyncoC)\/.*$/`